utils/saasherder.py

```diff
@@ -389,15 +389,19 @@ def _collect_images(self, resource):
         return images
 
     @staticmethod
-    def _check_image(image, image_patterns, image_auth, error_prefix):
+    @retry()
+    def _is_image_valid(image, image_auth):
+        return Image(image, **image_auth)
```

Comment:
this is not checking is the image is valid. The check happens on bool(Image()), not on instance creation.

Reply:
thanks! changed to `return bool(Image(image, **image_auth))`. makes sense?

Reply:
yes!